packages/extension-polkagate/src/fullscreen/nft/utils/util.ts

```
@@ -0,0 +1,244 @@
+    }
+
+    /**
+     * Handles the difference between NFT and Unique metadata formats:
+     * In Unique metadata, the image URL is stored in the 'mediaUri' property.
+     * In standard NFT metadata, the image URL is stored in the 'image' property.
+     * Then it converts 'mediaUri' to 'image' if present, ensuring a consistent
+     * interface for the rest of the application to work with.
+     */
+    if ('mediaUri' in itemMetadata) {
+      itemMetadata.image = itemMetadata.mediaUri as string;
+      delete itemMetadata.mediaUri;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)